### PR TITLE
chore(docs): Add examples to the `Float64` module

### PR DESCRIPTION
**Pull request by spotandjake**
This improves the docs in the `Float64` module, It adds examples for the float64 syntax to the top following the pattern ive been using in other libs, Adds example to all the functions. and updates the documentation for `Float64.infinity` and `Float64.nan` to mention the alternative syntax.

### CODE REVIEW

**Review by alex-snezhko — APPROVED** [review]

File: stdlib/float64.gr [hunk 1]
```diff
@@ -22,6 +27,7 @@ from Numbers use {
 
 /**
  * Infinity represented as a Float64 value.
+ * This is equivalent to `Infinityd`.
```
Comment (phated) [inline stdlib/float64.gr:30]:
```suggestion
 * This is an alternative to the `Infinityd` literal.
```

File: stdlib/float64.gr [hunk 2]
```diff
@@ -30,6 +36,7 @@ provide let infinity = Infinityd
 
 /**
  * NaN (Not a Number) represented as a Float64 value.
+ * This is equivalent to `NaNd`.
```
Comment (phated) [inline stdlib/float64.gr:39]:
```suggestion
 * This is an alternative to the `NaNd` literal.
```

NaN is not equivalent to NaN

**Review by phated — CHANGES_REQUESTED** [review]
A little rewording

**Comment by spotandjake** [comment]
Made those changes

**Review by phated — APPROVED** [review]

**Pull request merged**